autoload/ale/cursor.vim

```diff
@@ -26,7 +26,7 @@ function! ale#cursor#TruncatedEcho(original_message) abort
 
         " The message is truncated and saved to the history.
         setlocal shortmess+=T
-        exec "norm! :echomsg l:message\n"
+        echomsg l:message
```

Comment:
Write `execute 'echomsg l:message'` here instead. One of the things the script tests for is people leaving stray echo lines lying around, and you can silence the warning by doing that.

Reply:
ok, I'll update. Thank you for explanation.

Reply:
I'm not fully understanding why the tailing `\n` is added. If `execute "echomsg l:message\n" is better, please let me know.

Reply:
For future reference: 

Quote: https://vim.fandom.com/wiki/Get_shortened_messages_from_using_echomsg

> The second step is tricky. :echo does not shorten long messages. :echomsg, by itself, does not shorten long messages. Only when :echomsg is used under :norm, are long messages truncated. And don't forget trailing \n when using :exe norm.

Removing this has resulted in many people complaining about "Press ENTER" messages on long diagnostic messages: #4141